index.src.html

```diff
@@ -610,87 +610,190 @@ <h2 id="mitigations">
     Mitigation Strategies
   </h2>
 
-  <h3 id="secure-contexts">
-    Secure Contexts
+  To mitigate the security and privacy risks you’ve identified in your
+  specification as you’ve filled out the questionnaire and consulted with PING,
+  you may want to apply one or more of the mitigations described below to your
+  feature.
+
+  <h3 id="data-minimization">
+    Data Minimization
   </h3>
 
-  In the presence of an <a>active network attacker</a>, offering a feature to
-  an insecure origin is the same as offering that feature to every origin (as
-  the attacker can inject frames and code at will). Requiring an encrypted and
-  authenticated connection in order to use a feature can mitigate this kind of
-  risk.
+  Minimization is a strategy that involves exposing as little information to
+  other communication partners as is required for a given operation to
+  complete. More specifically, it requires not providing access to more
+  information than was apparent in the user-mediated access or allowing the
+  user some control over which information exactly is provided.
+
+  For example, if the user has provided access to a given file, the object
+  representing that should not make it possible to obtain information about
+  that file's parent directory and its contents as that is clearly not what is
+  expected.
+
+  Basic fingerprinting guidance can be found here.
+
+  In context of data minimization it is natural to ask what data is passed
+  around between the different parties, how persistent the data items and
+  identifiers are, and whether there are correlation possibilities between
+  different protocol runs.
+
+  For example, the W3C Device APIs Working Group has defined the following
+  requirements in their Device API Privacy Requirements document.
+
+  *   APIs must make it easy to request as little information as required for
+      the intended usage. For instance, an API call should require specific
+      parameters to be set to obtain more information, and should default to
+      little or no information.
+  *   APIs should make it possible for user agents to convey the breadth of
+      information that the requester is asking for. For instance, if a developer
+      only needs to access a specific field of a user address book, it should be
+      possible to explicitly mark that field in the API call so that the user
+      agent can inform the user that this single field of data will be shared.
+  *   APIs should make it possible for user agents to let the user select,
+      filter, and transform information before it is shared with the requester.
+      The user agent can then act as a broker for trusted data, and will only
+      transmit data to the requester that the user has explicitly allowed.
+
+  Data minimization is applicable to specification authors, implementers as
+  well as to those deploying the final service.
```

Comment:
"is applicable to specification authors and implementers, as well as..."

Reply:
Thanks! This is addressed in 444a3f0.